app/screens/Login/Login.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { Text, View, Image, TextInput, TouchableHighlight } from 'react-native';
+
+import { iconImages } from './../../images';
+
+import { styles } from './styles';
+
+class Login extends Component {
+    state = {
+        inputTextEmail: 'email',
```

Comment:
Don't pass default text inside fields please. Or remove it on focus i.e.
better user placeholder for this purpose.
[placeholder](https://facebook.github.io/react-native/docs/textinput#placeholder)